Gulpfile.mjs

```diff
@@ -422,19 +418,20 @@ function buildRollupDts(packages) {
   const sourcemap = process.env.NODE_ENV === "production";
   return Promise.all(
     packages.map(async packageName => {
-      const input = `${packageName}/lib/index.d.ts`;
-      fancyLog(`Bundling '${chalk.cyan(input)}' with rollup ...`);
+      const input = `${mapToDts(packageName)}/src/index.d.ts`;
+      const output = `${packageName}/lib/index.d.ts`;
+      fancyLog(`Bundling '${chalk.cyan(output)}' with rollup ...`);
+
       const bundle = await rollup({
         input,
         plugins: [rollupDts()],
+        onwarn(warning, warn) {
+          if (warning.code !== "CIRCULAR_DEPENDENCY") warn(warning);
```

Comment:
It doesn't cause issues because types are hoisted. It's only a problem when two modules depend on each other and they export/import non-hoistable declarations:


```js
// a.js
import { b } from "b";
export let a = b;
```
```js
import { a } from "a";
export let b = a;
```